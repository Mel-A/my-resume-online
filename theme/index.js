const theme = {
  colors: {
    alabaster: '#FAFAFA',
    astral: '#327DA0',
    aquamarine: '#A1DAD7',
    azure: '#0080FF',
    black: '#000000',
    cobalt: '#003DB3',
    gray: '#333333',
    malibu: '#66B0FF',
    mercury: '#F7F7F7',
    onahau: '#CCE5FF',
    sail: '#80bdff',
    transparent: 'inherit',
    white: '#FFFFFF'
  },
  shadows: {
    codgray073: 'rgba(20, 20, 20, 0.73)',
    dovegray052: 'rgba(100, 100, 100, 0.52)',
    shaft07: 'rgba(39, 39, 39,0.7)',
    tundora04: 'rgba(70, 70, 70, 0.4)',
  },
  fonts: {
    small0: '10px',
    small4: '14px',
    small6: '16px',
    small8: '18px',
    medium0: '20px',
    medium2: '22px',
    medium4: '24px',
    medium6: '26px',
    medium8: '28px',
    big0: '30px'
  }
}

export default theme