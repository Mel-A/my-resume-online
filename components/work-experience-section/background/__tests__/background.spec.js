import { shallow, mount } from 'enzyme'
import Background from '..'
import { ThemeProvider } from "styled-components"
import theme from '../../../../theme'

jest.mock('react-i18next', () => ({
  useTranslation: () => ({t: key => key})
}))

describe('Background', () => {
  it('background renders correctly', () => {
    const background = shallow(<Background />)
    expect(background).toMatchSnapshot()
    
  })

  it('background renders theme correctly', () => {
    const background = mount(<Background />, {
      wrappingComponent: ThemeProvider,
      wrappingComponentProps: {
        theme,
      },
    })
    expect(background).toMatchSnapshot()
  })
})
