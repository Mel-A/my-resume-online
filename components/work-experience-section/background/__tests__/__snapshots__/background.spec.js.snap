// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Background background renders correctly 1`] = `ShallowWrapper {}`;

exports[`Background background renders theme correctly 1`] = `ReactWrapper {}`;
